#4940 Fix for build on Mac

sosnovsky opened the pull request:
This PR fixes extension building in macOS

close #4940

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

rrrooommmaaa commented:
Hi @sosnovsky
I'm having the `bak` files piling up after each build. Perhaps, it'd be better to delete them?
![image](https://user-images.githubusercontent.com/23335944/217582001-9fd1e6e7-f5d3-4137-946a-013a9436c424.png)

sosnovsky commented:
@rrrooommmaaa I updated build script to not produce `.bak` files in new PR - https://github.com/FlowCrypt/flowcrypt-browser/pull/4945
macOS and Linux have a bit different syntaxes for `sed` command, so I just added conditional check for the current OS
